src/views/network/CreateNetwork.vue

```diff
@@ -18,24 +18,24 @@
 <template>
   <div class="form-layout">
     <a-tabs defaultActiveKey="1" :animated="false">
-      <a-tab-pane :tab="$t('label.isolated')" key="1" v-if="this.isAdvancedZoneWithoutSGAvailable()">
-        <CreateIsolatedNetworkForm
+      <a-tab-pane :tab="$t('label.l2')" key="1">
```

Comment:
@davidjumani can the first network/tab be isolated, then l2 and then shared? Most people would want to see isolated by default; also this is reused in the deploy VM form.

Reply:
It can, but then l2 form pops open first since the if condition on the isolated form takes time to evaluate, and the reordering of the tabs once evaluated looks messy